### DISCUSSION TITLE
MTV Taiwan

### DISCUSSION THREAD

**Discussion by GiovanniSebastian** [iptv-org/community#680] [Q&A]
**Channel Name:** MTV Taiwan
**Channel ID:** MTV.tw
**Country:** Republic of China (Taiwan)
**Language:** Mandarin Chinese
**Website:** https://www.mtv.com.tw/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19678
